### DISCUSSION TITLE
SEGY output request

### DISCUSSION THREAD

**Discussion by vkazei** [DASDAE/dascore#428] [Ideas]
Would greatly help standardize output between vendors for subsequent seismic processing. Few of us at the DFOS workshop hope to see a SEGY output option soon :)

**Comment by d-chambers** [comment]
Hey @vkazei,

It's good to hear from you. I hope its starting to cool off a bit in Houston. 

I agree with you. I am personally not a big fan of the format, and it does make me a little sad to think DASCore would be adding more SEGY files to world, but so many programs use SEGY as inputs that its utility is undeniable. ProdML output is also high on our priority list. 

@aaronjgirard added the SEGY read support, I wonder if he has any thoughts on a road map for adding SEGY write support? We current just use segyio under the hood so, at first glance, I think it would be relatively straight-forward to produce SEGY files but I do know there is a lot of nuance to the format so it might not be as easy as it appears. 

Would you be willing to help us test the outputs to make sure they play nicely with your processing workflows?

**Reply by vkazei** [reply]
Hi @d-chambers,

Sure, I'd be happy to provide some feedback on usage. Here is an open DAS VSP dataset for example:
https://researchdata.edu.au/walk-away-vertical-experiment-geolab/1915113
maybe we could test some DASCore tools and save updated segys?

**Comment by aaronjgirard** [comment]
Hi Vladimir and Derrick,

I could put in the effort to generate SEGY outputs, but one of the biggest
issues was ensuring that all the header information is correct, since the
DAS files don't have all of the SEGY information. If you could give some
input on what the most important SEGY file headers you'd need would be, I
can look into that soon. SEGY output hasn't been a priority lately, but now
that you've brought it up it could be done. Thanks for the file and I will
look into this

Best,
*Aaron Girard*

**Comment by d-chambers** [comment]
Hey @vkazei and @aaronjgirard, there is now segy write support in the latest DASCore version (0.1.5). It is really basic currently, so we fully anticipate needing to improve it, but you can take it for a spin and let us know what's missing or where it doesn't work.
